client/src/screens/MyTeamsScreen/MyTeamsScreen.tsx

```diff
@@ -1,18 +1,58 @@
 import Header from "../../components/Header/Header";
 import TeamRow from "../../components/TeamRow/TeamRow";
+import TeamCard from "../../components/TeamCard/TeamCard";
 
 const MyTeamsScreen = () => {
-  const dummyData = {
-    name: "Juan Pablo",
-    lastName: "Perez Duran",
-    email: "jppd@ibm.mx",
-    role: "Front-End Developer",
+  const dummyData = [
```

Comment:
same here this dummy should go in Container and not be named like this

Reply:
all dummies will be moved on the next PR, it will be a mess if I do it right now